### DISCUSSION TITLE
Controlling qutebrowser

### DISCUSSION THREAD

**Discussion by SebastianMoyano** [qutebrowser/qutebrowser#6308] [Q&A] [answered]
hi,

Is there any way to control qutebrowser like when using selenium? as it is very lightweight it would be very interesting if it can be done.

Thanks in advance!

**Comment by The-Compiler** [comment, marked as answer]
According to some stuff I found ([one](https://github.com/cisco-open-source/qtwebdriver/issues/31), [two](https://forum.qt.io/topic/96202/unrecognized-chrome-version-when-using-selenium-python-bindings-and-chromedriver)) it might be possible to use Selenium's chromedriver with QtWebEngine - I've never tried though, and I think it'd be more beneficial to use QtWebEngine directly (which is even more minimal) rather than qutebrowser for this.
